texthero/representation.py

```diff
@@ -894,3 +895,74 @@ def meanshift(
 """
 
 # TODO.
+
+"""
+Normalization.
+"""
+
+
+def normalize(s: pd.Series, norm="l2") -> pd.Series:
+    """
+    Normalize every cell in a Pandas Series.
+
+    Input has to be a Representation Series.
+
+    Parameters
+    ----------
+    s: Pandas Series
```

Comment:
s: Pandas Series, RepresentationSeries (this is not that important yet but will become after we merge #69)

The sentence "Input has to be a Representation Series." is probably not strictly necessary and for consistency, we should either add it in every function or remove it